net.pioneerspacesim.Pioneer.json

```diff
@@ -0,0 +1,102 @@
+{
+	"app-id": "net.pioneerspacesim.Pioneer",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "1.6",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "pioneer",
+
+	"finish-args": [
+		"--share=ipc",
+		"--socket=x11",
+		"--socket=wayland",
+		"--socket=pulseaudio",
+		"--device=dri",
+		"--persist=.pioneer"
+	],
+
+	"build-options" : {
+		"cflags": "-O3",
+		"cxxflags": "-O3",
+		"strip": true,
+		"env": {
+			"PIONEER_DATA_DIR": "/app/usr/share/pioneer"
+		}
+	},
+
+	"modules": [
+		{
+			"name": "sigcpp",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "http://ftp.acc.umu.se/pub/GNOME/sources/libsigc++/2.10/libsigc++-2.10.0.tar.xz",
+					"sha256": "f843d6346260bfcb4426259e314512b99e296e8ca241d771d21ac64f28298d81"
+				}
+			]
+		},
+		{
+			"name": "assimp",
+			"buildsystem": "cmake",
+			"config-opts": [
+				"-DASSIMP_BUILD_ALL_IMPORTERS_BY_DEFAULT:BOOL=NO",
+				"-DASSIMP_BUILD_COLLADA_IMPORTER:BOOL=YES",
+				"-DASSIMP_BUILD_OBJ_IMPORTER:BOOL=YES",
+				"-DASSIMP_BUILD_3MF_IMPORTER:BOOL=YES",
+				"-DASSIMP_BUILD_ASSIMP_TOOLS:BOOL=NO",
+				"-DASSIMP_BUILD_TESTS:BOOL=NO"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://github.com/assimp/assimp/archive/v4.1.0.tar.gz",
+					"sha256": "3520b1e9793b93a2ca3b797199e16f40d61762617e072f2d525fad70f9678a71"
+				}
+			]
+		},
+		{
+			"name": "libglu",
```

Comment:
It's available in `shared-modules` repository that can be used as git submodule.